examples/appstore_permissions/main.tf

```diff
@@ -1,13 +1,14 @@
 terraform {
   required_providers {
     lumos-appstore = {
-      source = "lumos.com/tf/lumos-appstore"
+      source = "lumos.com/tf/lumos"
     }
   }
 }
 
 provider "lumos-appstore" {
   api_token = "<API TOKEN>"
+  base_url="<BASE_URL>"
```

Comment:
I think we should remove this in the example, this is only useful for Lumos internal work for now. Only in the future if we do subdomains e.g. `flexport.lumos.com` would this be useful to a customer